common4j/src/main/com/microsoft/identity/common/java/commands/InteractiveTokenCommand.java

```diff
@@ -79,7 +81,23 @@ public AcquireTokenResult execute() throws Exception {
 
                 span.setAttribute(AttributeName.controller_name.name(), controller.getClass().getSimpleName());
 
-                return controller.acquireToken((InteractiveTokenCommandParameters) getParameters());
+                final AcquireTokenResult result = controller.acquireToken((InteractiveTokenCommandParameters) getParameters());
+
+                if (result == null) {
+                    span.setStatus(StatusCode.ERROR, "empty result");
```

Comment:
Could "empty result" and "empty exception" be set as static strings in BaseCommand and referenced from there in both classes?

Reply:
We can put it there, but for now I've merged the PR in as I have a few PRs to merge in. I can address this in the future.